parse/release-proguard.pro

```diff
@@ -1,7 +1,13 @@
+-keep @com.parse.ParseClassName class com.parse.*
 -keepnames class com.parse.** { *; }
+-keepclassmembers public class * extends com.parse.** {
+   public <init>(...);
+}
 
 # Required for Parse
 -keepattributes *Annotation*
 -keepattributes Signature
+
 # https://github.com/square/okio#proguard
 -dontwarn okio.**
+-dontwarn org.codehaus.mojo.animal_sniffer.*
```

Comment:
I assume this was added by mistake, or does the SDK have auch a dependency?

Reply:
Done, thanks!